scalafix/2.13/rules/src/main/scala/fix/Collectionstrawman_v0.scala

```diff
@@ -86,9 +86,20 @@ case class Collectionstrawman_v0(index: SemanticdbIndex)
         removeTokensPatch + replaceCommasPatch
     }.asPatch
 
+  val copyToBuffer = SymbolMatcher.normalized(
+    Symbol("_root_.scala.collection.TraversableOnce.copyToBuffer.")
+  )
+
+  def replaceCopyToBuffer(ctx: RuleCtx): Patch =
+    ctx.tree.collect {
+      case t @ q"${copyToBuffer(Term.Select(collection, _))}($buffer)" =>
```

Comment:
Pro tip, you can put the `collection` qualifier like this to avoid Term.Select
```scala
@ val q"$qual.${q"foo"}(..$arg)" = q"qual.foo(1, 2)"
qual: Term = Term.Name("qual")
arg: List[Term] = List(Lit.Int(1), Lit.Int(2))

```

People have different tastes however, I personally prefer Term.Select :)